src/components/ProjectForm.js

```diff
@@ -0,0 +1,491 @@
+          });
+
+        this.setState({'project_name': ''});
+        this.setState({'project_image': ''});
+        this.setState({'project_description': ''});
+        this.setState({'project_goals': ''});
+        this.setState({'project_needs': ''});
+        this.setState({'project_origins': ''});
+        this.setState({'project_status': ''});
+        this.setState({'project_org_description': ''});
+        this.setState({'project_holy_goals': ''});
+        this.setState({'project_github': ''});
+        this.setState({'project_slack': ''});
+        this.setState({'project_trello': ''});
+        this.setState({'project_email': ''});
+        this.setState({'project_video': ''});
+      }
+
+
+
+      render() {
+        const {
+          project_name,
+          project_image,
+          project_description,
+          project_goals,
+          project_needs,
+          project_origins,
+          project_status,
+          project_org_description,
+          project_holy_goals,
+          project_video,
+          project_github,
+          project_slack,
+          project_trello,
+          project_email
+                } = this.state;
+        const isEnabled = project_name.length > 0
+                && project_image.length > 0
+                && project_description.length > 0
+                && project_goals.length > 0
+                && project_needs.length > 0
+                && project_origins.length > 0
+                && project_status.length > 0
+                && project_org_description.length > 0
+                && project_holy_goals.length > 0;
+
+        return (
+
+          <form onSubmit={this.onSubmit.bind(this)}>
+          <div className="columns is-centered is-multiline">
+
+          {/* NAME */}
+          <div className="column is-3">
+            <br/>
+            <label className="label is-medium">Project Name</label>
+            <div className="field is-horizontal required">
+              <div className="field-body">
+                <div className="field">
+                  <div className="control">
+                    <input className="input"
+                      type="text"
+                      name="project_name"
+                      value={project_name}
+                      onChange={this.onChange}
+                      placeholder="What is the name of your project?" />
+                      <p className="help">This field is required</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+
+          {/* IMAGE UPLOAD */}
+          <div className="column is-2 is-centered">
+            <br/>
+            <label class="label is-medium">Upload an image...</label>
+            <div className='buttons fadein'>
+            <div className='button'>
+              <label htmlFor='single'></label>
+              <input type='file' id='single' name="project_image" value={project_image} onChange={this.onChange} />
```

Comment:
This input is colliding with adjacent elements. How about putting it on a separate line?

Reply:
I just pushed changes that reordered the elements on the line so that the image upload is on the far right. Putting the image upload on its own line made the page have a lot of extra white space and even though the order is a little weird, overall I think it looks better this way. Let me know if you think it needs to be changed.